### DISCUSSION TITLE
How to narrow down the visible nodes with a predefined 'filter set'

### DISCUSSION THREAD

**Discussion by theelectricsoul** [freeplane/freeplane#1799] [Help]
Hello

# Scenario

1. I have a bunch of nodes with different styles. 
2. I filter the nodes that have the style 'task' only. 
3. I activate the function 'Applied to filtered nodes', because I want to further narrow down the nodes by adding more filter criteria.
![image](https://github.com/freeplane/freeplane/assets/10260832/8a94a27e-0aae-4529-8e20-cf8333a30b55)
4. I pick a pre-definded filter to hopefully narrow down the results.
![image](https://github.com/freeplane/freeplane/assets/10260832/b20ca312-2b6b-4e2f-b9ca-d9384c1ed1c0)

# Result
Picking a predefined filter ignores the currently filtered nodes. Freeplane simply applies the pre-defined filter to all nodes on the map.

# Question
Is there a way to use pre-defined filters to further narrow down the filter?

*I know that I could simply create a pre-defined filter that combines all my needed filter criteria, however I do not want to create a filter for every needed filter combination that I might use.*

**Comment by dpolivaev** [comment]
I need to check it. If it does not work as you expect,  it might be a bug.

**Reply by dpolivaev** [reply]
https://github.com/freeplane/freeplane/issues/1800

**Reply by theelectricsoul** [reply]
Wait...   I am getting funky results.. I need to check my logic.

I close this post
